csrc/runtime/executor.cpp

```diff
@@ -745,11 +745,15 @@ void KernelExecutor::initializeExecutorEntry(
       auto out_val = compiled_kernel_->kernel()->outputs()[output_idx];
       NVF_ERROR(out_val->isA<TensorView>(), "Output is not a TensorView");
       info.tv = out_val->as<TensorView>();
-      NVF_ERROR(
-          !info.tv->hasAllocation(),
-          "Accepting allocated outputs is not currently supported with allocation domain. ",
-          "Allocation domain found for tv: ",
-          info.tv->toString());
+      if (info.tv->hasAllocation()) {
```

Comment:
I don't think the added test exercises this code path. Can you create a new test or make sure this code path is covered?

Reply:
The code path is covered--in the test below the output TensorView has an allocation and hits this path.